dss-core/src/main/java/io/github/ztkmkoo/dss/core/message/rest/DssRestExceptionHandlerCommandRequest.java

```diff
@@ -0,0 +1,21 @@
+package io.github.ztkmkoo.dss.core.message.rest;
+
+import io.github.ztkmkoo.dss.core.actor.rest.service.DssRestActorService;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+public class DssRestExceptionHandlerCommandRequest implements DssRestExceptionHandlerCommand {
+    private static final long serialVersionUID = 5752260184738368572L;
+
+    private final DssRestActorService service;
```

Comment:
It looks transient(not serializable)?
If yes, please add the keyword

Reply:
Thank you for review! 

I couldn't understand your opinion. Could you explain more detail?

Reply:
transient is a variables modifier used in serialization. At the time of serialization, if we don’t want to save value of a particular variable in a file, then we use transient keyword. When JVM comes across transient keyword, it ignores original value of the variable and save default value of that variable data type.


The code may be
``` java
public class DssRestExceptionHandlerCommandRequest implements DssRestExceptionHandlerCommand {
    private static final long serialVersionUID = 5752260184738368572L;

    private transient final DssRestActorService service;
```